README.md

```diff
@@ -10,21 +11,61 @@ We are using [backport project](https://backports.wiki.kernel.org/index.php/Main
  We will add a new branch ubuntu/ubuntu<x.y> whenever a version is deprecated or moved to the maintenance phase.
   
 # Supported Version/kernel
-  our current backport is based on **Ubuntu® 20.04 (linux-oem kernel 5.14)**. We are using the header of the latest available kernel at the time of backporting. However, it may not be compatible with the latest version at the time of installation.
+  Our current backport is based on **Ubuntu® 20.04 (linux-oem kernel 5.14)**. We are using the header of the latest available kernel at the time of backporting. However, it may not be compatible with the latest version at the installation time.
   Please refer [Version](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/ubuntu/main/versions)
   file to check the BASE_KERNEL_NAME. It will point to the kernel version which is being used during backporting.
 
+  In case of an issue with the latest kernel, please install the kernel version pointed by BASE_KERNEL_NAME.
+
+    sudo apt install linux-headers-5.14.0-1033-oem \
+    linux-image-5.14.0-1033
```

Comment:
1034 appears to be the latest.  Is 1033 correct?

Reply:
kernel information should be taken from https://github.com/intel-gpu/intel-gpu-i915-backports/blob/ubuntu/main/versions, this is the kernel version that is being tested during release.
 and it will keep changing with every release.